IFComp/lib/IFComp/Schema/Result/Comp.pm

```diff
@@ -302,5 +303,43 @@ sub get_vote_counts_from_non_unique_ips {
     return $data;
 }
 
+sub emails {
+    my $self            = shift;
+    my $is_disqualified = 0;
```

Comment:
Why is this here as a separate var, instead of just putting 0 into the query? Was it going to be a method parameter at some point?